### PR TITLE
BERI-738: Fixed log4j

### PR DESCRIPTION
**Pull request by patryk-beekeeper**
Upgraded gradle (4.0 -> 6.7.1) and java (8 -> 11).
Added beekeeper-gradle-plugin
Updated io.beekeeper:beekeeper-chatbot-sdk (2.4.0 -> 3.7.1), otherwise It seems log4j 1.2.17 is still present. 

Any hints how to test this?

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,18 +1,13 @@
...
 
 LABEL Description="Meeting Bot"
-RUN apk add --update bash curl unzip && rm -rf /var/cache/apk/*
 
 ENV "BKPR_SERVER_DOCKER_MODE"="true"
 
 ADD build/distributions/bkpr-spotify-to-youtube-bot-*.tar /opt/bkpr/
 RUN mv /opt/bkpr/bkpr-spotify-to-youtube-bot-* /opt/bkpr/service/
 
-RUN curl -q -L -C - -b "oraclelicense=accept-securebackup-cookie" -o /tmp/jce_policy-8.zip -O http://download.oracle.com/otn-pub/java/jce/8/jce_policy-8.zip \
```
Comment (patryk-beekeeper) [inline Dockerfile:12]: I think it's not needed for java11

File: build.gradle [hunk 2]
```diff
@@ -10,39 +12,38 @@ version '1.0'
...
 repositories {
     mavenCentral()
     maven { url 'https://s3-eu-west-1.amazonaws.com/beekeeper-marketplace-sdk/maven2/release/' }
 }
 
 dependencies {
-    compile 'io.beekeeper:beekeeper-sdk:2.4.0'
-    compile 'io.beekeeper:beekeeper-chatbot-sdk:2.4.0'
-    testCompile group: 'junit', name: 'junit', version: '4.12'
+    implementation 'io.beekeeper:beekeeper-sdk:3.7.1'
```
Comment (patryk-beekeeper) [inline build.gradle:23]: Just adding plugin is not enough in my opinion:

image

Comment (kasatch) [reply build.gradle:23]: Just a test comment

Comment (uvifian) [reply build.gradle:23]: ...

**Comment by uvifian** [comment]
Approved

**Comment by HappyTetrahedron** [comment]
to test this, deploy it and then chat to "Spotybot" on hive ^^ (if it doesn't work, oh well, it's not a priority)

**Comment by uvifian** [comment]
Spotybot is not answering to me without deployment, so it seems already broken. Just saying

**Comment by HappyTetrahedron** [comment]
Guess it fell into disrepair then. Feel free to just completely remove it if that makes things easier

**Pull request merged**